src/FetchProvider.ts

```diff
@@ -40,7 +39,7 @@ export default class FetchProvider extends BaseConnection implements Provider {
       });
       return await response.json();
     } catch (error) {
-      switch(this.getEnv()) {
+      switch(Networking.getEnv()) {
       case "NODE": {
         if (error instanceof FetchError) {
           throw new ResolutionError(ResolutionErrorCode.NamingServiceDown, {
```

Comment:
Why are we saying that a naming service is down if it's a `FetchError`? It has lots of other reasons why this error can occur, besides a provider being actually down.

Reply:
I can't comment the lines below, but 
```
        if (error.name === "TypeError") {
          throw new ResolutionError(ResolutionErrorCode.NamingServiceDown, {
            method: this.name,
          });
        }
```

makes even less sense to me